content/general/concepts/tech-stacks/tech-stacks.md

```diff
@@ -0,0 +1,82 @@
+## Tech Stacks
+
+A technology stack is a set of technologies stacked together to build any application. Technology stacks are commonly referred to as technology infrastructures or solutions stacks, which are essential for building easy-to-maintain, scalable web applications.
+
+A tech stack refers to tools, programming languages, and technologies that work together to build digital products or solutions such as websites, mobile, and web apps.
+
+### Frontend Tech Stack
+The front end is the part of a software program that a user interacts with. One of the primary goals of front-end development is to create a smooth or frictionless user experience. Basically, the user interface of an application or website should be simple and straightforward.
+
+There are three major elements that make up the frontend stack. These are
+
+- HTML (Hypertext Markup Language): HTML is the standard markup language for building Web pages. It is also the building block of the web and is used to structure a web page and its content.
+
+- CSS (Cascading style sheets): CSS is the language we use to style an HTML document. It describes how HTML elements should be displayed.
+
+- Javascript: JavaScript is the programming language of the Web. It is used to make web pages interactive and it allows the implementation of dynamic web pages. Some of the popular javascript frameworks include React, Vue, Angular and svelte.
+
+### Backend Tech Stack
+The back end also known as the server-side refers to parts of a computer application or a program's code that allow it to operate and that cannot be accessed by a user. The back end of a computer system is where the majority of data and operating syntax are kept and accessible.
+
+The backend tech stack comprises the following elements:
+  Programming Languages: A programming language is a computer language programmers use to develop software programs or other sets of instructions for computers to execute. Examples include Javascript, Python, Java, etc.
+
+- Frameworks: A framework is a tool that provides ready-made components or solutions that are customized in order to speed up development. Popular frameworks used in development include Laravel, Ruby on Rails, Express, and Django.
+
+- Databases: A database is a systematic collection of structured information or data. The most common databases include MongoDB, Postgresql, MySQL, and SQL.
+
+- Servers: A server is a computer program or device that provides a service to another computer program and its user, also known as the client. It responds and accepts to requests made over a network. Commonly used servers are AWS, Google Cloud, Apache, Nginx, Microsoft's Internet Information Services (IIS), etc.
```

Comment:
Any chance we could move/incorporate this information into the existing general [back-end entry](https://www.codecademy.com/resources/docs/general/back-end)?